### PR TITLE
feat: add login to did auth proxy

### PR DESCRIPTION
**Pull request by hejkerooo**
Before reviewing please check - https://github.com/energywebfoundation/dsb-client-gateway/pull/103 

Feature list:

- [x] Add DID Auth Proxy login,
- [x] Extract auth. from dsb module

Todo:
- [x] Handle refresh token
- [x] Handle expired tokens
- [ ] Tests

### CODE REVIEW

File: apps/dsb-client-gateway-api/src/app/modules/dsb-client/service/dsb-api.service.ts [hunk 1]
```diff
@@ -312,6 +294,42 @@ export class DsbApiService implements OnModuleInit {
     }
   }
 
+  protected async initExtChannel(): Promise<void> {
+    try {
+      const { data } = await promiseRetry(async (retry, attempt) => {
+        return lastValueFrom(
+          this.httpService.post(
+            'https://aemovc.eastus.cloudapp.azure.com/channel/initExtChannel',
```
Comment (vikaskumr) [inline apps/dsb-client-gateway-api/src/app/modules/dsb-client/service/dsb-api.service.ts:302]: Can we please put this URl in .env file

**Review by vikaskumr — APPROVED** [review]
Please resolve one small comment rest everything looks good

File: apps/dsb-client-gateway-api/src/app/modules/dsb-client/service/dsb-api.service.ts (hunk 1 above)
Comment (hejkerooo) [reply apps/dsb-client-gateway-api/src/app/modules/dsb-client/service/dsb-api.service.ts:302]: I will make it as env. variable once we get rid of all unused endpoints from previous DSB, does it sound alright for you?

Comment (vikaskumr) [reply apps/dsb-client-gateway-api/src/app/modules/dsb-client/service/dsb-api.service.ts:302]: ok that sound good

**Pull request merged**